react/typings/events.d.ts

```diff
@@ -218,7 +218,9 @@ export interface AddToWishlistData extends EventData {
   event: 'addToWishlist'
   eventType: 'vtex:addToWishlist'
   eventName: 'vtex:addToWishlist'
-  product: Product
+  items: {
+    product: Product
```

Comment:
Would it still work if we replaced `Product` with [`ProductSummary`](https://github.com/vtex-apps/google-tag-manager/blob/e0b0a82677619d82799001255cae0bdc859a4c10/react/typings/events.d.ts#L473)? 

`ProductSummary` has a `sku` field. 
![CleanShot 2023-05-08 at 19 34 59](https://user-images.githubusercontent.com/381395/236952288-8fe2d22a-76df-4bef-9f3b-300d3c6facdb.png)

That way, we wouldn't need [to add `sku` to `Product`](https://github.com/vtex-apps/google-tag-manager/pull/141/files#diff-791619406f30fb7292cdecb798d2a840d54a926399fdcd740a7706dbe852c9a6R461).
![CleanShot 2023-05-08 at 19 34 11](https://user-images.githubusercontent.com/381395/236952202-67827678-b734-422d-a217-5a33447bf475.png)